api/src/main/java/com/datastrato/gravitino/authorization/SecurableObject.java

```diff
@@ -73,6 +74,22 @@ public interface SecurableObject {
    */
   Type type();
 
+  /**
+   * The privileges of the securable object. For example: If the securable object is a table, the
+   * privileges could be `READ TABLE`, `WRITE TABLE`, etc. If a schema has the privilege of `LOAD
+   * TABLE`. It means the role can all tables of the schema.
+   *
+   * @return The privileges of the role.
+   */
+  List<Privilege> privileges();
```

Comment:
Are these privileges inheritable in your current design?

Reply:
They are not inheritable.